_includes/en/guides/400_ci_cd_workflow/040_github_actions.md.liquid

```diff
@@ -59,51 +58,71 @@ werf generates stages on the basis of the git history. So, if there is no histor
 ```
 {% endraw %}
 
-The action `werf/actions/converge` combines all the necessary steps, sets up an environment, and invokes the appropriate command.
+The action `werf/actions/install` perform werf installation.
+
+{% raw %}
+```yaml
+- name: Install
+  uses: werf/actions/install@v2
+```
+{% endraw %}
+
+Step `Converge` sets up an environment, and invokes the appropriate command.
 
 {% raw %}
 ```yaml
 - name: Converge
-  uses: werf/actions/converge@v2
-  with:
-    env: ANY_ENV_NAME
-    kube-config-base64-data: ${{ secrets.KUBE_CONFIG_BASE64_DATA }}
+  run: |
+    . $(werf ci-env github --as-file)
+    werf converge
   env:
     WERF_SET_ENV_URL: "envUrl=ANY_ENV_URL"
+    WERF_ENV: ANY_ENV_NAME
+    WERF_KUBE_CONFIG_BASE64: ${{ secrets.KUBE_CONFIG_BASE64_DATA }}
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 {% endraw %}
 
+> Environment variables **`WERF_REPO,`** **`GITHUB_TOKEN`** and **`WERF_REPO_GITHUB_TOKEN`** for converge should only be used if building images is required otherwise they can be omitted 
```

Comment:
The `WERF_REPO_GITHUB_TOKEN` environment variable is not mentioned here, and the user needs it only for the container registry cleanup.